koinos/chain/error.proto

```diff
@@ -26,6 +26,7 @@ enum error_code {
     unknown_hash_code            = -101;
     unknown_dsa                  = -102;
     unknown_system_call          = -103;
+    no_operation                 = -104;
```

Comment:
Not sure what this new error is for but we've been using `internal_error` for these:

```c++
   const auto* transaction = context.get_transaction();
   KOINOS_ASSERT( transaction != nullptr, internal_error_exception, "transaction does not exist" );

   *ret.mutable_value() = get_nested_field_value( context, *transaction, field );
```

Reply:
The reason for a different error code is that it is possible for user code to easily call `get_operation` when there is no operation via the authorize overrides. `internal_error` is not appropriate because the error is not the result of a bug in the framework, but an error in the contract. We have a similar set of logic with `field_not_found` when calling `get_transaction_field`. It seemed appropriate that a missing operation follow that logic as it is a user error, rather an an internal one.